autocomplete/src/PickCountry.jsx

```diff
@@ -0,0 +1,111 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+import "./App.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+function PickCountry({
+  selectableCountries = [],
+  includeFlags = true,
+  onSubmit = () =>
+    console.warn("Forgot to set onSubmit in PickCountry component"),
+}) {
+  const [countries, setCountries] = useState([
+    {
+      name: "Loading...",
+      flag: "🔄",
+    },
+  ]);
+
+  const [selectedCountry, setSelectedCountry] = useState();
+
+  useEffect(() => {
+    if (
+      Array.isArray(selectableCountries) &&
+      selectableCountries.length === 0
+    ) {
+      axios
+        .get("https://restcountries.com/v3.1/all", {
+          params: {
+            fields: `name${includeFlags ? ",flag" : ""}`,
+          },
+        })
+        .then((response) => {
+          setCountries(
+            response.data.map((country) => ({
+              name: country.name.common,
+              flag: country.flag,
+            })),
+          );
+        });
+    } else {
+      if (!Array.isArray(selectableCountries)) {
+        setCountries([
+          {
+            name: "Invalid selectableCountries Input",
+            flag: "⚠️",
+          },
+        ]);
+        console.warn("selectableCountries received a non-array input.");
+      }
+      // An array of strings
```

Comment:
Instead of adding comments, use code to describe what you are trying to do.


```js
if (onlyStrings(selectableCountries)) {
```

or 

```js
if (allOptionsAreStrings) {
```